### ISSUE TITLE
Could not find host mac address for 0.0.0.0

### ISSUE THREAD

**Issue by jeroenlandheer** [kubernetes-sigs/sig-windows-tools#159]
**Describe the bug**
kube-proxy v1.21.x crashes with the message: 

```
F0729 15:22:54.388647    3828 server.go:489] unable to create proxier: unable to create ipv4 proxier: Could not find host mac address for 0.0.0.0, hostname: ****-k8s-w3, clusterCIDR : 10.244.0.0/16, nodeIP:0.0.0.0
```

**To Reproduce**
- Upgrade from 1.20.9 to 1.21.0

**Expected behavior**
No error message

**Kubernetes (please complete the following information):**
 - Windows Server version:  Microsoft Windows [Version 10.0.17763.1577] (1809)
 - Kubernetes Version: 1.21.3
 - Docker: 20.10.0
 - Flannel: 0.12.0

**Comment by dfateyev** [comment]
Kube-proxy-windows is broken starting from 1.21.0.
The workaround is to downgrade worker to 1.20: use v1.20.0 in `kube-proxy.yml` and `PrepareNode.ps1` call.

**Comment by jsturtevant** [comment]
This is tracked in k/k: https://github.com/kubernetes/kubernetes/issues/100966

fixed by https://github.com/kubernetes/kubernetes/pull/101047 with a backport in progress to 1.21: https://github.com/kubernetes/kubernetes/pull/103783

The work around for 1.21 is https://github.com/kubernetes-sigs/sig-windows-tools/pull/147

**Comment by dfateyev** [comment]
@jsturtevant any chances that the fix will land 1.21.4? (since automatic merge to release-1.21 above is still open)

**Comment by jeroenlandheer** [comment]
It looks like in the latest release this has been fixed. Thanks :)
